onshapeAPI.js

```diff
@@ -1 +1,298 @@
-// hello
+const fetch = require('node-fetch');
+const crypto = require('crypto');
+const url = require('url');
+const querystring = require('querystring');
+const path = require('path');
+const keys = require('../config/apikey');
+
+const gltfConfiguration = {
+    includeExportIds: false,
+    formatName: 'GLTF',
+    flattenAssemblies: false,
+    yAxisIsUp: false,
+    triggerAutoDownload: false,
+    storeInDocument: false,
+    connectionId: '',
+    versionString: '',
+    grouping: true,
+    destinationName: '',
+    configuration: 'default',
+    cloudStorageAccountId: null,
+    emailLink: false,
+    emailTo: null,
+    emailSubject: null,
+    emailMessage: null,
+    sendCopyToMe: null,
+    passwordRequired: null,
+    password: null,
+    validForDays: null,
+    fromUserId: null,
+    linkDocumentWorkspaceId: keys.workspaceId,
+    resolution: 'high',
+    distanceTolerance: 0.00012,
+    angularTolerance: 0.1090830782496456,
+    maximumChordLength: 10
+};
+
+// Request Creation Function
+const buildHeaders = function (method, path, stream, querystring) {
+    let headers = {
+        'Content-Type': 'application/json',
+        'On-Nonce': crypto.randomBytes(25).toString('hex'),
+        'Date': (new Date()).toUTCString(),
+        'Accept': stream ? 'application/vnd.onshape.v1+octet-stream' : 'application/vnd.onshape.v1+json'
+    };
+    const queryString = querystring ? querystring : '';
+    const hmacString = (`${method}\n${headers['On-Nonce']}\n${headers['Date']}\n${headers['Content-Type']}\n${path}\n${queryString}\n`).toLowerCase();
+    let hmac = crypto.createHmac('sha256', keys.secretKey);
+    hmac.update(hmacString);
+    let signature = hmac.digest('base64');
+    const authorization = `On ${keys.accessKey}:HmacSHA256:${signature}`;
+    headers['Authorization'] = authorization;
```

Comment:
Remove that constant and assign right in to the headers like so: 
```suggestion
   headers['Authorization'] = `On ${keys.accessKey}:HmacSHA256:${signature}`;
```
You don't use this constant anywhere else in code so it's redundant.